client/src/components/FlightBarogramm.vue

```diff
@@ -182,6 +152,12 @@ const options = {
       legend: {
         display: false,
       },
+      crosshair: {
+        line: {
+          color: "#GGG", // crosshair line color
```

Comment:
Sieht man im Dark Theme eigentlich fast garnicht mehr

#08556d wäre das XCCup Blau. Das kann man besser erkennen.

Reply:
Ich bastel gerade in nem anderen branch schon weiter, da ist das korrigiert